src/Mbstring/bootstrap.php

```diff
@@ -55,7 +55,7 @@ function mb_detect_encoding($string, $encodings = null, $strict = false) { retur
     function mb_detect_order($encoding = null) { return p\Mbstring::mb_detect_order($encoding); }
 }
 if (!function_exists('mb_parse_str')) {
-    function mb_parse_str($string, &$result = []) { parse_str($string, $result); }
+    function mb_parse_str($string, &$result = []): bool { parse_str($string, $result); return (bool) $result; }
```

Comment:
the return type should be removed: the native function doesn't have one until PHP8, see https://3v4l.org/m49P0